dali/operators/decoder/nvjpeg/decoupled_api/nvjpeg_decoder_decoupled_api.h

```diff
@@ -695,6 +699,21 @@ class nvJPEGDecoder : public Operator<MixedBackend>, CachedDecoderImpl {
   static constexpr int kOutputDim = 3;
 
  private:
+  /**
+   * @brief Resets task priority count
+   */
+  void ResetTaskPrioritySeq() {
+    task_priority_seq_ = 0;
+  }
+
+  /**
+   * @brief Gets the next task priority to ensure FIFO execution in the thread pool (descencing integers)
+   */
+  int64_t GetTaskPrioritySeq() {
```

Comment:
A bit weird with all the wrappers for task_seq--, but maybe it's better that way.

Reply:
removed